packages/core/admin/admin/src/types/adminConfiguration.ts

```diff
@@ -0,0 +1,107 @@
+import type { Attribute, Schema } from '@strapi/types';
+
+/* -------------------------------------------------------------------------------------------------
+ * Configuration – This should be exported from the Content Manager plugin.
```

Comment:
@joshuaellis I have copied these types from i18 plugin as inject column was written for locales. And I am exporting it from admin to be utilised in other plugins.

Reply:
Great! You'll need to do a fair bit of refactoring to get this working in V5, the types are exported, but the data structures are quite different 🤓 but either way sounds good!!